pkg/apis/storage/types.go

```diff
@@ -291,9 +292,29 @@ type CSIDriverSpec struct {
 	// "csi.storage.k8s.io/pod.name": pod.Name
 	// "csi.storage.k8s.io/pod.namespace": pod.Namespace
 	// "csi.storage.k8s.io/pod.uid": string(pod.UID)
+	// "csi.storage.k8s.io/ephemeral": "true" iff the volume is an ephemeral inline volume
```

Comment:
This is ["iff == if (and only if)"](https://en.wikipedia.org/wiki/If_and_only_if) - I've replaced it with "if" because the "otherwise" in the next line is enough to avoid the ambiguity that usually comes from using just "if".